spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-lattice/src/main/java/org/springframework/cloud/data/module/deployer/lattice/ReceptorLrpModuleDeployer.java

```diff
@@ -39,9 +39,9 @@
  * @author Patrick Peralta
  * @author Mark Fisher
  */
-public class ReceptorModuleDeployer implements ModuleDeployer {
+public class ReceptorLrpModuleDeployer implements ModuleDeployer {
```

Comment:
Maybe we can drop "Receptor" from the name now that "Lrp" is a qualifier (and it's in the "lattice" package).